### DISCUSSION TITLE
Adding include field in query filters

### DISCUSSION THREAD

**Discussion by fyrepenguin** [factly/dega#644] [Ideas]
When we're querying data for posts, categories, or any other field we're getting data from other tables for media, claims, categories, tags, users, ratings, claimants, formats, etc

since we might not need all those fields every time we make the query or every theme that we develop, if we add another filter called `include` or `add` which takes those fields in there like below and then we make the calls to the respective tables depending on those from the API to DB, I think we might be able to increase performance and decrease the load on the DB as well.
```
posts(includes: ["claims","medium"])
```

**Comment by deshetti** [comment]
@fyrepenguin Is that a normal pattern in GraphQL?

**Comment by fyrepenguin** [comment]
@deshetti, I noticed that in a REST API of ghost CMS [https://ghost.org/docs/content-api/#include](here), not sure about it in GraphQL and need to check with @shreeharsha-factly on performance benefits
